types/highland/highland-tests.ts

```diff
@@ -19,6 +19,7 @@ var anyArr: any[];
 var strArr: string[];
 var numArr: string[];
 var funcArr: Function[];
+var regExp = /a/
```

Comment:
It looks like line 17 already defines`exp: RegExp`, maybe that variable can be reused?